src/components/PromotedActionsBar.tsx

```diff
@@ -81,11 +80,11 @@ const PromotedActions = {
             const topmostCentralPaneRoute = getTopmostCentralPaneRoute(navigationRef.getRootState() as State<RootStackParamList>);
 
             if (topmostCentralPaneRoute?.name !== SCREENS.SEARCH.CENTRAL_PANE && isTextHold) {
-                ReportUtils.changeMoneyRequestHoldStatus(reportAction, ROUTES.REPORT_WITH_ID.getRoute(reportAction?.childReportID ?? ''));
+                ReportUtils.changeMoneyRequestHoldStatus(reportAction, backTo);
                 return;
             }
 
-            ReportUtils.changeMoneyRequestHoldStatus(reportAction, ROUTES.SEARCH_REPORT.getRoute(reportAction?.childReportID ?? ''));
+            ReportUtils.changeMoneyRequestHoldStatus(reportAction, backTo);
```

Comment:
pass `backTo` with `ROUTES.REPORT_WITH_ID` will change the behaviour here because this should be `ROUTES.SEARCH_REPORT` not `ROUTES.REPORT_WITH_ID`

Reply:
what do you think if we pass some params like `ReportID` other than `backTo`?